app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/SearchRequestAutoMapperProfile.cs

```diff
@@ -77,6 +77,17 @@ public SearchRequestAutoMapperProfile()
         }
     }
 
+    public class IssueByTypeConverter : IValueConverter<int?, string>
+    {
+        public string Convert(int? source, ResolutionContext context)
+        {
+            if (source == InformationSourceType.ICBC.Value) return InformationSourceType.ICBC.Name;
```

Comment:
you can replace that with a linq expression `Enumeration.GetAll<InformationSourceType>().FirstOrDefault(x => x.Name == ...`